DigitalLearningSolutions.Data/DataServices/UserDataService/UserCentreDetailsDataService.cs

```diff
@@ -60,14 +60,19 @@ ELSE 0
             if (detailsAlreadyExist)
             {
                 connection.Execute(
-                    @"UPDATE UserCentreDetails
-                    SET Email = @email
-                    WHERE userID = @userId AND centreID = @centreId",
+                    string.IsNullOrWhiteSpace(email)
+                        ? @"UPDATE UserCentreDetails
+                        SET Email = @email,
+                            EmailVerified = null
+                        WHERE userID = @userId AND centreID = @centreId"

```

Comment:
I'm tempted to say we should set the email to NULL as well when we're in this email is null or whitespace case since a whitespace email is invalid.
Also can you capitalise the NULL for consistency